bindings/rust/s2n-tls-hyper/src/connector.rs

```diff
@@ -110,14 +115,22 @@ where
 pub struct Builder<Http, ConnBuilder> {
     http: Http,
     conn_builder: ConnBuilder,
+    insecure_http: bool,
 }
 
 impl<Http, ConnBuilder> Builder<Http, ConnBuilder> {
+    /// Allows communication with insecure HTTP endpoints in addition to secure HTTPS endpoints.
+    pub fn with_insecure_http(&mut self) -> &mut Self {
```

Comment:
I've seen some preference for having a boolean argument to these kind of builder methods, to make toggling it off and on slightly easier. So something like `pub fn with_insecure_http(&mut self, enabled: bool)`

Reply:
That makes sense!